tests/integration/utils/call-cli.ts

```diff
@@ -17,11 +17,16 @@ export const callCli = async function (
   parseJson = false,
 ): // eslint-disable-next-line @typescript-eslint/no-explicit-any
 Promise<any> {
-  const { stdout } = await execa.node(cliPath, args, {
+  const { stdout, stderr } = await execa.node(cliPath, args, {
     timeout: CLI_TIMEOUT,
     nodeOptions: [],
     ...execOptions,
   })
+  if (process.env.DEBUG_TESTS) {
```

Comment:
Urelated but this is so annoying. We have this `DEBUG_TESTS` mechanism that surfaces spawned process logs but it wasn't being leveraged in the `callCli()` integration test helper case for some reason.